Sming/Services/SpifFS/spiffs_sming.c

```diff
@@ -67,14 +73,14 @@ bool spiffs_format_internal(spiffs_config *cfg)
   sect_last = flashmem_get_sector_of_address(sect_last);
   debugf("sect_first: %x, sect_last: %x\n", sect_first, sect_last);
   ETS_INTR_LOCK();
-  int total = sect_last - sect_first;
+  int total = sect_last - sect_first + 1;
```

Comment:
This is not correct (see [PR 1313](https://github.com/SmingHub/Sming/pull/1313#issuecomment-476843308))

Reply:
As you can see ([here](https://github.com/SmingHub/Sming/pull/1313/commits/43a9a4f719fefba4921b573b7c69633243605b4d)) I also did the same change but in conjunction with the change in line 67 (which is the fix of the bug).